bin/configs/python-fastapi.yaml

```diff
@@ -5,3 +5,4 @@ templateDir: modules/openapi-generator/src/main/resources/python-fastapi
 sourceFolder: "src"
 additionalProperties:
   hideGenerationTimestamp: "true"
+  disallowAdditionalPropertiesIfNotPresent: true
```

Comment:
@wing328 In the `bin/configs/python-fastapi.yaml`, as in almost all other sample configs, I added `disallowAdditionalPropertiesIfNotPresent: true` to avoid a PR that has massive changes across all examples.  I expected that the samples would have 0 changes as a result, because `true` was the default value prior to this PR.